### ISSUE TITLE
Tower Ghost is unequipping exotic class items

### ISSUE THREAD

**Issue by edmiller47** [dasilva333/TowerGhostForDestiny#661]
When equipping an exotic armor piece (helmet, chest, etc.), Tower Ghost is incorrectly unequipping class items that have "The Life Exotic" perk. 

Version 3.9.2.4

**Comment by dasilva333** [comment]
Thanks I'll look into that

**Comment by dasilva333** [comment]
@edmiller47 Try it on 3.9.2.7 I haven't made any changes but I tested it and it seems to be working fine. Please test it again (using the "Light" button to equip to max light) to make sure it should be working as expected. Let me know thanks.

**Comment by edmiller47** [comment]
Richard, I've never ever used the "Light" button. I reported this bug
because I noticed that when I equip an exotic armor piece (e.g. Ruin Wings)
while an exotic class item was already equipped, TG unequipped the class
item before equipping the armor piece. It shouldn't.

Just tried repeating this on 3.9.2.7, and the behavior is unchanged. This
just started happening with one of the recent updates, definitely wasn't
happening before.

Plus, there's no guarantee the exotic piece will be max light. Tried the
"Light" button anyway and didn't have a test case because the exotic piece
is not the highest light level, and so it didn't get equipped.
